### PR TITLE
Make the DNN model compatible with CPU & Update the Estimator doc

### PR DESCRIPTION
**Pull request by bxdd**
## Description
This pr aims to make the DNN model compatible with CPU and update the `Estimator` doc.

## Motivation and Context
The pr is related to #47 

## How Has This Been Tested?
- [ ] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [x] If you are adding a new feature, test on your own test scripts.

## Types of changes

- [ ] Fix bugs
- [x] Add new feature
- [x] Update documentation

### CODE REVIEW

File: docs/component/estimator.rst [hunk 1]
```diff
@@ -574,10 +574,24 @@ The `qlib_data` field describes the parameters of qlib initialization.
       region: "cn"
     
 - `provider_uri`
-    The local directory where the data loaded by 'get_data.py' is stored.
+    Type: str. The URI of the Qlib data. For example, it could be the location where the data loaded by ``get_data.py`` are stored.
 - `region`
-    - If region == ``qlib.config.REG_CN``, 'qlib' will be initialized in US-stock mode. 
-    - If region == ``qlib.config.REG_US``, 'qlib' will be initialized in china-stock mode.
+    - If `region` == "us", ``Qlib`` will be initialized in US-stock mode. 
+    - If `region` == "cn", ``Qlib`` will be initialized in china-stock mode.
```
Comment (you-n-g) [inline docs/component/estimator.rst:580]: The concerete value in is upper case [here](https://github.com/microsoft/qlib/blob/main/qlib/config.py#L64)
I think change all of them to lower case will be better

File: qlib/contrib/model/pytorch_nn.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def __init__(
         self,
         input_dim,
         output_dim,
-        layers=(256, 256, 128),
+        layers=(256, 512, 768, 1024, 768, 512, 256, 128, 64),
```
Comment (you-n-g) [inline qlib/contrib/model/pytorch_nn.py:50]: Will the change of NN structure affects the performance greatly?

File: qlib/contrib/model/pytorch_nn.py [hunk 3]
```diff
@@ -133,11 +136,14 @@ def __init__(
         )
 
         self._fitted = False
-        self.dnn_model.cuda()
+
```
Comment (you-n-g) [inline qlib/contrib/model/pytorch_nn.py:139]: PEP8: too many blanks

Comment (you-n-g) [reply qlib/contrib/model/pytorch_nn.py:139]: @Derek-Wds   It seems that we have run the pep8 checker but the errors are not raised.
image

We use [black ](https://github.com/psf/black)with max line width set to 120 as our pep8 checker.

File: qlib/contrib/model/pytorch_nn.py [hunk 4]
```diff
@@ -316,20 +330,20 @@ def update(self, val, n=1):
 
 
 class Net(nn.Module):
-    def __init__(self, input_dim, output_dim, layers=(256, 256, 256), loss="mse"):
+    def __init__(self, input_dim, output_dim, layers=(256, 512, 768, 512, 256, 128, 64), loss="mse"):
         super(Net, self).__init__()
         layers = [input_dim] + list(layers)
         dnn_layers = []
-        drop_input = nn.Dropout(0.1)
+        drop_input = nn.Dropout(0.05)
```
Comment (you-n-g) [inline qlib/contrib/model/pytorch_nn.py:337]: Will these changes of structure affect the NN performance greatly?

File: qlib/contrib/model/pytorch_nn.py (hunk 2 above)
Comment (bxdd) [reply qlib/contrib/model/pytorch_nn.py:50]: Yes

**Pull request merged**